core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -36,16 +37,20 @@ angular.module('oppia').directive('imageUploader', [
         onFileChanged: '=',
         errorMessage: '@',
         width: '@',
-        getAllowedImageFormats: '&allowedImageFormats'
+        getAllowedImageFormats: '&allowedImageFormats',
+        backgroundWhileUploading: '<'
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/forms/custom-forms-directives/' +
         'image-uploader.directive.html'),
       link: function(scope: ImageUploaderCustomScope, elt) {
         var onDragEnd = function(e) {
           e.preventDefault();
-          $('.image-uploader-drop-area').removeClass(
-            'image-uploader-is-active');
+          scope.backgroundWhileUploading = false;
+          // Since we are using JQlite for event handling
+          // which will run outside angular
+          // that's why we should trigger scope.$apply() Manually.
```

Comment:
```suggestion
          // JQlite is used for for event handling
          // in this case and it runs outside angular.
          // Due to this, scope.$apply() has to be triggered manually
          // for the changes to be reflected.
```

I have a question, is it jQlite or jQuery that is used here?